plugins/modules/mso_schema_template_contract_filter.py

```diff
@@ -334,11 +337,16 @@ def main():
             ops.append(dict(op='replace', path=contract_path + '/filterType', value=contract_ftype))
             ops.append(dict(op='replace', path=contract_path + '/scope', value=contract_scope))
 
-        mso.existing['displayName'] = contract_display_name if contract_display_name is not None else contract_obj.get('displayName')
+        if contract_display_name:
```

Comment:
tried to use the same function to replace this part, but got some errors. 
trying to find out whether it's feasible